index.html

```diff
@@ -17,9 +17,9 @@
         <nav class="menu">
             <img src="./assets/icons/Union.png" alt="Menu Icon" class="header-image-mobile">
             <ul class="top-nav-desktop">
-                <li class="top-nav-items-desktop">Portfolio</li>
-                <li class="top-nav-items-desktop">About</li>
-                <li class="top-nav-items-desktop">Contact</li>
+                <a href="#" class="top-nav-items-desktop"><li>Portfolio</li></a>
+                <a href="#" class="top-nav-items-desktop"><li>About</li></a>
+                <a href="#" class="top-nav-items-desktop"><li>Contact</li></a>
```

Comment:
- [ ] Good implement the changes asked on previous review but you should keep the `<li>` in the `<ul>` like that 
example : 
```HTML
 <ul>
        <li>
           <a href="#" >Portfolio</a>
        </li>
        <li>
          <a href="#" >About</a>
         </li>
         <li>
           <a href="#" >Contact</a>
         </li>
</ul>

```